.github/workflows/e2e_ios.yml

```diff
@@ -22,9 +27,21 @@ env:
   APP_CLIENT_SECRET: ${{ secrets.APP_CLIENT_SECRET }}
   # IAM production app client secret
   APP_CLIENT_SECRET_PROD: ${{ secrets.APP_CLIENT_SECRET_PROD }}
+  # Slack API token used by fastlane
+  SLACK_API_TOKEN: ${{ secrets.SLACK_API_TOKEN }}
 
 jobs:
+  start_slack_thread:
+    if: github.event_name == 'schedule'
+    name: Start Slack thread
+    uses: ./.github/workflows/start_slack_thread.yml
+    secrets: inherit
+    with:
+      channel_name: va-mobile-build-alerts
+      message: 'E2E iOS build process starting. Please see :thread: for results. This process may take a while.'
```

Comment:
Had a similar suggestion for the messaging here too
https://github.com/department-of-veterans-affairs/va-mobile-app/pull/7199#discussion_r1408135342

Reply:
updated